src/main/java/org/cactoos/scalar/Or.java

```diff
@@ -121,6 +122,22 @@
         );
     }
 
+    /**
+     * Ctor.
+     * @param subject The subject
+     * @param conditions Funcs to map
+     * @param <X> Type of items in the iterable
+     */
+    @SafeVarargs
+    public <X> Or(final X subject, final Func<X, Boolean>... conditions) {
+        this(
+            new Mapped<>(
+                item -> (Scalar<Boolean>) () -> item.apply(subject),
```

Comment:
We do not use casts in oop

Reply:
We should use the cast because java cannot determine which functional method is this. We also used the same casting in the same class.